test/Sentry.Tests/Protocol/TransactionTests.cs

```diff
@@ -0,0 +1,37 @@
+using FluentAssertions;
+using Sentry.Extensibility;
+using Sentry.Internal;
+using Sentry.Protocol;
+using Xunit;
+
+namespace Sentry.Tests.Protocol
+{
+    public class TransactionTests
+    {
+        [Fact]
+        public void Serialization_Roundtrip_Success()
+        {
+            // Arrange
+            var transaction = new Transaction(DisabledHub.Instance, null)
+            {
+                Name = "my transaction",
+                Operation = "some op",
+                Description = "description",
+                Request = new Request {Method = "GET", Url = "https://test.com"},
```

Comment:
For examples the top level example domains such as example.com or e.g. sentry.example should always be used:

https://tools.ietf.org/html/rfc2606#page-2